erizo_controller/erizoClient/src/utils/ConnectionHelpers.js

```diff
@@ -39,6 +39,14 @@ const GetUserMedia = (config, callback = () => {}, error = () => {}) => {
         screenConfig.video.mandatory = config.video.mandatory || {};
         screenConfig.video.mandatory.chromeMediaSource = 'desktop';
         screenConfig.video.mandatory.chromeMediaSourceId = config.desktopStreamId;
+        if (config.audio) {
+          screenConfig.audio = {
+            mandatory: {
+              chromeMediaSource: 'system',
```

Comment:
does this work? afaik electron didn't support the use of audio, but I might be wrong.

Reply:
I never tested on Electron.js but according to the [documentation](https://electronjs.org/docs/api/desktop-capturer):

> To capture both audio and video from the entire desktop the constraints passed to navigator.mediaDevices.getUserMedia must include chromeMediaSource: 'desktop', for both audio and video, but should not include a chromeMediaSourceId constraint.

So it needs to be fixed in the PR.